README.md

```diff
@@ -0,0 +1,89 @@
+# Purpose of this Repository:
+This repository is intended as a template to standardize and ease the initialization process for Technical Reports from members of the Advanced Reactors and Fuel Cycles group at the University of Illinois at Urbana-Champaign. While this version of the template is geared towards an ARFC Member's report, it can be adapted to any individual's needs.

```

Comment:
```suggestion
This repository is intended as a template to standardize and ease the initialization 
process for Technical Reports from members of the Advanced Reactors and Fuel 
Cycles group at the University of Illinois at Urbana-Champaign. While this version 
of the template is geared towards an ARFC Member's report, it can be adapted to 
any individual's needs.

```